tasks/winbuildscripts/chocopack.bat

```diff
@@ -0,0 +1,11 @@
+if not exist c:\mnt\ goto nomntdir
+
+@echo c:\mnt found, continuing
+
+Powershell -C "C:\mnt\tasks\winbuildscripts\Generate-Chocolatey-Package.ps1"
```

Comment:
I think we need to add `|| exit 1` at the end of this line, so that when the powershell script fails then this script also fails and we get a red CI. Can you verify what's the result status of this script when the powershell script fails and fix it if needed?